doc/api/assert.md

```diff
@@ -1575,3 +1575,4 @@ argument.
 [Strict Equality Comparison]: https://tc39.github.io/ecma262/#sec-strict-equality-comparison
 [enumerable "own" properties]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Enumerability_and_ownership_of_properties
 [prototype-spec]: https://tc39.github.io/ecma262/#sec-ordinary-object-internal-methods-and-internal-slots
+[`process.on('exit')`]: process.html#process_event_exit
```

Comment:
Links at the bottom are sorted in ASCII order. So I think this should be moved up above line 1568, just under `assert.throws()`.